README.md

```diff
@@ -164,3 +170,48 @@ l * incorrect_above + (1-l) * correct_below
 certain threshold and `correct_below` describes the fraction of correct 
 predictions below a certain threshold. Threshold and `l` can be configured 
 through environment variables.
+
+# Github Action
+
+> Take note of Github Action's [usage limit](https://help.github.com/en/actions/getting-started-with-github-actions/about-github-actions#usage-limits)
+of 360 minutes per job. Keep this in mind when choosing `max_evals`.
+
+## Inputs
+
+### search_space
```

Comment:
should we organize these as list or table as well?

Reply:
yup